Makefile

```
@@ -169,6 +169,12 @@ $(LLVM_BUILDDIR)/build.ninja: llvm-source
 $(LLVM_BUILDDIR): $(LLVM_BUILDDIR)/build.ninja
 	cd $(LLVM_BUILDDIR); ninja $(NINJA_BUILD_TARGETS)
 
+# Build Binaryen
+.PHONY: binaryen
+binaryen: build/wasm-opt
+build/wasm-opt:
+	cd lib/binaryen; cmake -G Ninja . -DBUILD_STATIC_LIB=ON; ninja
```

```suggestion
	cd lib/binaryen && cmake -G Ninja . -DBUILD_STATIC_LIB=ON && ninja
```

Not sure whether this works as-is in a Makefile but generally not continuing on errors is a good thing.